packages/server/migrations/1595162567466-migration.ts

```diff
@@ -1,22 +1,31 @@
 import { MigrationInterface, QueryRunner } from 'typeorm';
 
 export class migration1595162567466 implements MigrationInterface {
-    name = 'migration1595162567466'
+  name = 'migration1595162567466';
 
-    public async up(queryRunner: QueryRunner): Promise<void> {
-        await queryRunner.query('CREATE TABLE "email_reset" ("id" SERIAL NOT NULL, "userId" character varying NOT NULL, "emailResetToken" character varying NOT NULL, "expiresIn" bigint NOT NULL, CONSTRAINT "PK_c7cc1d8ec099867c393d29adafb" PRIMARY KEY ("id"))');
-        await queryRunner.query('CREATE TABLE "email_verification" ("id" SERIAL NOT NULL, "userId" character varying NOT NULL, "emailVerificationToken" character varying NOT NULL, "expiresIn" bigint NOT NULL, CONSTRAINT "PK_b985a8362d9dac51e3d6120d40e" PRIMARY KEY ("id"))');
-        await queryRunner.query('CREATE TABLE "health_check" ("id" SERIAL NOT NULL, CONSTRAINT "PK_bb6ae6b7bca4235bf4563eaeaad" PRIMARY KEY ("id"))');
-        await queryRunner.query('CREATE TABLE "password_reset" ("id" SERIAL NOT NULL, "userId" character varying NOT NULL, "passwordResetToken" character varying NOT NULL, "expiresIn" bigint NOT NULL, CONSTRAINT "PK_8515e60a2cc41584fa4784f52ce" PRIMARY KEY ("id"))');
-        await queryRunner.query('CREATE TABLE "user" ("id" SERIAL NOT NULL, "firstName" character varying NOT NULL, "lastName" character varying NOT NULL, "email" character varying NOT NULL, "password" character varying NOT NULL, "status" character varying NOT NULL, "facebookId" character varying, CONSTRAINT "PK_cace4a159ff9f2512dd42373760" PRIMARY KEY ("id"))');
-    }
-
-    public async down(queryRunner: QueryRunner): Promise<void> {
-        await queryRunner.query('DROP TABLE "user"');
-        await queryRunner.query('DROP TABLE "password_reset"');
-        await queryRunner.query('DROP TABLE "health_check"');
-        await queryRunner.query('DROP TABLE "email_verification"');
-        await queryRunner.query('DROP TABLE "email_reset"');
-    }
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      'CREATE TABLE "email_reset" ("id" SERIAL NOT NULL, "userId" character varying NOT NULL, "emailResetToken" character varying NOT NULL, "expiresIn" bigint NOT NULL, CONSTRAINT "PK_c7cc1d8ec099867c393d29adafb" PRIMARY KEY ("id"))'
+    );
```

Comment:
Let's keep these compact, I kind of prefer them like this.
I also just fixed eslint indentation for all packages, this should help avoid fixes related to indentations, let's also merge that and keep that in mind:
https://github.com/dubbha/croton/pull/80